src/meson.build

```diff
@@ -0,0 +1,111 @@
+src_files = files(
+    'trg-cell-renderer-speed.c',
+    'trg-cell-renderer-counter.c',
+    'trg-cell-renderer-size.c',
+    'trg-cell-renderer-ratio.c',
+    'trg-cell-renderer-eta.c',
+    'trg-cell-renderer-priority.c',
+    'trg-cell-renderer-wanted.c',
+    'trg-cell-renderer-file-icon.c',
+    'trg-cell-renderer-epoch.c',
+    'trg-cell-renderer-numgteqthan.c',
+    'torrent-cell-renderer.c',
+    'trg-remote-prefs-dialog.c',
+    'trg-torrent-props-dialog.c',
+    'trg-torrent-add-url-dialog.c',
+    'trg-torrent-add-dialog.c',
+    'trg-torrent-move-dialog.c',
+    'trg-preferences-dialog.c',
+    'trg-stats-dialog.c',
+    'trg-about-window.c',
+    'trg-destination-combo.c',
+    'trg-state-selector.c',
+    'trg-general-panel.c',
+    'trg-torrent-graph.c',
+    'icons.c',
+    'trg-toolbar.c',
+    'trg-menu-bar.c',
+    'trg-status-bar.c',
+    'trg-file-parser.c',
+    'trg-json-widgets.c',
+    'trg-model.c',
+    'trg-sortable-filtered-model.c',
+    'trg-file-rename-dialog.c',
+    'trg-files-tree.c',
+    'trg-files-model.c',
+    'trg-files-tree-view-common.c',
+    'trg-files-tree-view.c',
+    'trg-files-model-common.c',
+    'trg-trackers-model.c',
+    'trg-trackers-tree-view.c',
+    'trg-peers-model.c',
+    'trg-peers-tree-view.c',
+    'trg-torrent-model.c',
+    'trg-torrent-tree-view.c',
+    'trg-persistent-tree-view.c',
+    'trg-tree-view.c',
+    'util.c',
+    'hig.c',
+    'bencode.c',
+    'trg-prefs.c',
+    'remote-exec.c',
+    'trg-gtk-app.c',
+    'requests.c',
+    'torrent.c',
+    'session-get.c',
+    'json.c',
+    'trg-client.c',
+    'trg-main-window.c',
+    'main.c',
+    'upload.c'
+)
+
+#print config file
+configure_file(output: 'config.h', configuration: conf_data)
+
+
+# man pages
+pod2man = find_program('pod2man', required: true)
+manpages = custom_target('manpages',
+                         input: '@0@.pod'.format(project_name),
+                         output: '@0@.1'.format(project_name),
+                         command: [pod2man, '--release=@0@'.format(project_version),
+                                   '--center="Transmission Remote GTK"',
+                                   '@INPUT@', '@OUTPUT@'],
+                         install: true,
+                         install_dir: get_option('mandir'))
+
+
+# gresources.xml
+gnome = import('gnome')
+gresource_data = meson.project_source_root() / 'data'
```

Comment:
You have a mix of `join_path()` and `/` throughout. I don't really have a preference but consistency is nice.

Reply:
That's fair, `/` is less characters so I guess I'll do that one